Gateway/Response/ConsumerMessageHandler.php

```diff
@@ -3,11 +3,25 @@
 namespace Buckaroo\Magento2\Gateway\Response;
 
 use Buckaroo\Magento2\Gateway\Helper\SubjectReader;
-use Buckaroo\Transaction\Response\TransactionResponse;
+use Magento\Framework\Message\ManagerInterface as MessageManager;
 use Magento\Payment\Gateway\Response\HandlerInterface;
 
-class ConsumerMessageHandler extends AbstractResponseHandler implements HandlerInterface
+class ConsumerMessageHandler implements HandlerInterface
```

Comment:
where is this class used and why?

Reply:
It is used in <virtualType name="SepaDirectDebitHandler"> and set the success message on success page with what we receive from plaza

Reply:
why not use the MessageManager directly ?